### PR TITLE
Replace checkboxes in list items with `check-list-item`

### PR DESCRIPTION
**Pull request by bramkragten**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [X] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/components/ha-check-list-item.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { css } from "lit";
+import { CheckListItem } from "@material/mwc-list/mwc-check-list-item";
+import { customElement } from "lit/decorators";
+
+const styles = [
```
Comment (balloob) [inline src/components/ha-check-list-item.ts:5]: Why define above and not use `static styles = […]` ?

File: src/components/ha-form/ha-form-multi_select.ts [hunk 2]
```diff
@@ -179,6 +210,9 @@ export class HaFormMultiSelect extends LitElement implements HaFormElement {
         display: block;
         pointer-events: none;
       }
+      mwc-check-list-item {
+        --mdc-theme-secondary: var(--primary-color);
+      }
```
Comment (balloob) [inline src/components/ha-form/ha-form-multi_select.ts:215]:
```suggestion
```

File: src/components/ha-radio.ts [hunk 3]
```diff
@@ -1,11 +1,19 @@
 import { Radio } from "@material/mwc-radio";
```
Comment (balloob) [inline src/components/ha-radio.ts:1]: Should we go all fancy and import just the base and the styles? [like this](https://github.com/improv-wifi/sdk-js/blob/main/src/components/ib-button.ts)

**Review by balloob — DISMISSED** [review]

File: src/components/ha-radio.ts (hunk 3 above)
Comment (bramkragten) [reply src/components/ha-radio.ts:1]: 👍🏼 yes

**Review by balloob — APPROVED** [review]

**Pull request merged**